### PR TITLE
Whishlist

### PR DESCRIPTION
**Pull request by harshitpaliwal95**
- Add: wishlist page
- responsive ☑️
Screenshot 2022-02-13 at 5 33 21 PM

Screenshot 2022-02-13 at 5 33 54 PM

### CODE REVIEW

File: wishlist-managment/wishlist.html [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+                <div class="card-btn-footer">
+                  <button class="btn btn-outline">Add to Cart</button>
+                </div>
+              </div>
+              <div class="card-component">
+                <div class="card-comp-img">
+                  <img
+                    class="card-top-img"
+                    src="../assets/nike-pegasus.webp"
+                    alt="loading"
```
Comment (shubhmehra) [inline wishlist-managment/wishlist.html]: alt messages can be improved

File: wishlist-managment/wishlist.html [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
```
Comment (shubhmehra) [inline wishlist-managment/wishlist.html]: same here

**Review by shubhmehra — APPROVED** [review]
Alt messages can be fixed!
otherwise good to go!

File: wishlist-managment/wishlist.html (hunk 1 above)
Comment (harshitpaliwal95) [reply wishlist-managment/wishlist.html]: resolve now

File: wishlist-managment/wishlist.css [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+}
+.main-product {
+  margin: auto;
+}
+.grid-three {
+  gap: 5rem;
+}
+/* card badge color */
+.card-badge {
+  background-color: black;
```
Comment (teja538) [inline wishlist-managment/wishlist.css:19]: suggestion: I think you can use variables for colors

**Review by teja538 — COMMENTED** [review]
everything seems good

File: wishlist-managment/wishlist.css (hunk 3 above)
Comment (harshitpaliwal95) [reply wishlist-managment/wishlist.css:19]: will use it

**Pull request merged**